From 8c12948661a15189af1b41a800697b0648b63c73 Mon Sep 17 00:00:00 2001
From: Gao Pan <b54642@freescale.com>
Date: Mon, 26 Oct 2015 12:58:25 +0800
Subject: [PATCH] MLK-11779 sim: imx: set gpio active high

Sim card CVCC is determined by the gpio value from sim controller.
How the CVCC is controlled on both post card boards is decribed below.

NCN8025:Vcc=ACTIVE_HIGH?5V:3V
TDA8035:Vcc=ACTIVE_HIGH?5V:1.8V

Different sim cards have different CVCC range. To support all cards
with same dts, this patch set gpio active high.

Signed-off-by: Gao Pan <b54642@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dts b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
index 4fecb18..c1623c9 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
@@ -269,7 +269,11 @@
 	assigned-clocks = <&clks IMX6UL_CLK_SIM_SEL>;
 	assigned-clock-parents = <&clks IMX6UL_CLK_SIM_PODF>;
 	assigned-clock-rates = <240000000>;
-	pinctrl-assert-gpios = <&gpio4 23 GPIO_ACTIVE_LOW>;
+	/* GPIO_ACTIVE_HIGH/LOW:sim card voltage control
+	 * NCN8025:Vcc = ACTIVE_HIGH?5V:3V
+	 * TDA8035:Vcc = ACTIVE_HIGH?5V:1.8V
+	 */
+	pinctrl-assert-gpios = <&gpio4 23 GPIO_ACTIVE_HIGH>;
 	port = <1>;
 	sven_low_active;
 	status = "okay";
-- 
1.8.0

